cmd/wire/main.go

```diff
@@ -66,11 +67,16 @@ func main() {
 		err = generate(os.Args[1:]...)
 	default:
 		fmt.Fprintln(os.Stderr, usage)
-		os.Exit(64)
+		exitCode = 64
 	}
 	if err != nil {
 		fmt.Fprintln(os.Stderr, "wire:", err)
-		os.Exit(1)
+		if exitCode == 0 {
+			exitCode = 1
+		}
+	}
+	if exitCode != 0 {
```

Comment:
This conditional isn't needed. `main()` reaching end of control flow is the same as `os.Exit(0)`.

Reply:
Done.